ams/server/src/main/java/com/netease/arctic/server/dashboard/controller/CatalogController.java

```diff
@@ -175,9 +197,9 @@ public void getCatalogTypeList(Context ctx) {
     ctx.json(OkResponse.of(catalogTypes));
   }
 
-  /** Convert server auth config to metaAuthConfig */
-  private Map<String, String> authConvertFromServerToMeta(
-      Map<String, String> serverAuthConfig, CatalogMeta oldCatalogMeta) {
+  /** fill server auth config to CatalogMeta */
```

Comment:
The comment here became meaningless to me as it is basically copied from the method name.

Reply:
I removed it.